addons/web_editor/static/src/js/editor/odoo-editor/src/utils/utils.js

```diff
@@ -2450,9 +2450,20 @@ export function enforceWhitespace(el, offset, direction, rule) {
     }
 }
 
-export function rgbToHex(rgb = '') {
+export function rgbToHex(rgb = '', bg = [255, 255, 255]) {
```

Comment:
The function was self explanatory before this change so we could do without a doc, but now there is no way for anyone else to understand what this second parameter is doing.

Reply:
True. I'll add a docstring and rename the argument.